doc/source/building/working_with_images/uninstall_package_requests.rst

```diff
@@ -0,0 +1,110 @@
+tools and development tools:
+
+.. code:: xml
+
+    <?xml version="1.0" encoding="utf-8"?>
+
+    <image schemaversion="6.8" name="openSUSE-Tumbleweed-container-image">
+      <description type="system">
+        <author>SUSE Containers Team</author>
+        <contact>containers@suse.com</contact>
+        <specification>openSUSE Tumbleweed container</specification>
+      </description>
+      <preferences>
+        <type image="docker">
+          <containerconfig
+            name="opensuse"
+            tag="tumbleweed"
+            maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"/>
+        </type>
+        <version>1.0.4</version>
+        <packagemanager>zypper</packagemanager>
+        <rpm-check-signatures>false</rpm-check-signatures>
+        <rpm-excludedocs>true</rpm-excludedocs>
+      </preferences>
+      <repository imageinclude="true" type="rpm-md" alias="OSS">
+        <source path="obs://openSUSE:Tumbleweed/standard"/>
+      </repository>
+      <packages type="image">
+        <package name="ca-certificates"/>
+        <package name="ca-certificates-mozilla"/>
+        <package name="coreutils"/>
+        <package name="iputils"/>
+        <package name="openSUSE-build-key"/>
+        <package name="krb5"/>
+        <package name="netcfg"/>
+        <package name="kubic-locale-archive"/>
+        <package name="make"/>
+        <package name="llvm-clang"/>
+        <archive name="foo_app_sources.tar.gz"/>
+      </packages>
+      <packages type="bootstrap">
+        <package name="aaa_base"/>
+        <package name="cracklib-dict-small"/>
+        <package name="filesystem"/>
+        <package name="openSUSE-release"/>
+        <package name="shadow"/>
+      </packages>
+      <!-- These packages will be uninstalled after running config.sh -->
+      <packages type="uninstall">
+        <package name="shadow"/>
+        <package name="make"/>
+        <package name="llvm-clang"/>
+      </packages>
+    </image>
+
+In this example after installing all the packages and archives, image
+repositories are configured and then the `config.sh` script is executed.
+In `config.sh` the `foo_app_sources.tar.gz` could be compiled using the
+`make` and `llvm` packages with something like a `make install` call. It is
+a common practice to build tiny and single purpose container images, thus
+makes sense to remove unneeded packages, like `make` and `llvm-clang`.
+In order to gracefully remove them, they have been included into the
+*type="uninstall"* packages list. Those packages will be removed including
+a dependency cleanup.
+
+.. warning::
+
+   Note that `uninstall` packages requests deletes the listed
+   packages, plus the packages dependent on the listed ones, plus any orphan
+   dependency of the listed packages. Use this feature with caution as it
+   can easily cause the removal of sensitive tools leading to failures in
+   later build stages.
```

Comment:
Hmn, I have some issues with this warning:

* plural vs. singular. If you stick with plural, it can't be "delete**s**". :wink: 
* it's a bit ambiguous to me. Does it mean to "use" the `uninstall` package requests? Or uninstalling package requests?
* Don't start with "Note" if it's a warning. That's a contradiction. :wink: 

One suggestion:

```
An `uninstall` packages request deletes:

* the listed packages,
* the packages dependent on the listed ones, and
* any orphaned dependecy of the listed packages.

Use this...
```